tests/Fixer/ArrayNotation/NoWhitespaceBeforeCommaInArrayFixerTest.php

```diff
@@ -27,14 +27,14 @@
      * @param string      $expected
      * @param null|string $input
      *
-     * @dataProvider testFixProvider
+     * @dataProvider providerFix
```

Comment:
What do you think about using a consistent naming for data providers?

### Prefix `provide`

```
$ git grep "@dataProvider provide[A-Z]"

doc/COOKBOOK-FIXERS.md:120:     * @dataProvider provideFixCases
doc/COOKBOOK-FIXERS.md:205:     * @dataProvider provideFixCases
tests/AutoReview/CommandTest.php:30:     * @dataProvider provideCommandHasNameConstCases
tests/AutoReview/DescribeCommandTest.php:30:     * @dataProvider provideDescribeCommandCases
tests/AutoReview/FixerTest.php:48:     * @dataProvider provideFixerDefinitionsCases
tests/AutoReview/FixerTest.php:137:     * @dataProvider provideFixerDefinitionsCases
tests/AutoReview/FixerTest.php:150:     * @dataProvider provideFixerDefinitionsCases
tests/AutoReview/FixerTest.php:163:     * @dataProvider provideFixerDefinitionsCases
tests/AutoReview/FixerTest.php:180:     * @dataProvider provideFixerConfigurationDefinitionsCases
tests/AutoReview/ProjectCodeTest.php:71:     * @dataProvider provideSrcConcreteClasses
tests/AutoReview/ProjectCodeTest.php:89:     * @dataProvider provideSrcClasses
tests/AutoReview/ProjectCodeTest.php:174:     * @dataProvider provideSrcClasses
tests/AutoReview/ProjectCodeTest.php:239:     * @dataProvider provideTestClasses
tests/AutoReview/ProjectCodeTest.php:254:     * @dataProvider provideTestClasses
tests/AutoReview/TransformerTest.php:32:     * @dataProvider provideTransformers
tests/Cache/CacheTest.php:156:     * @dataProvider provideCanConvertToAndFromJsonCases
tests/ConfigTest.php:195:     * @dataProvider provideRegisterCustomFixersCases
tests/Console/Command/FixCommandExitStatusCalculatorTest.php:35:     * @dataProvider provideCalculateCases
tests/Console/ConfigurationResolverTest.php:244:     * @dataProvider provideResolveConfigFileDefaultCases
tests/Console/ConfigurationResolverTest.php:473:     * @dataProvider provideResolveIntersectionOfPathsCases
tests/Console/ConfigurationResolverTest.php:991:     * @dataProvider provideDifferCases
tests/Console/Output/ErrorOutputTest.php:37:     * @dataProvider provideTestCases
tests/DocBlock/AnnotationTest.php:90:     * @dataProvider provideContent
tests/DocBlock/AnnotationTest.php:116:     * @dataProvider provideStartCases
tests/DocBlock/AnnotationTest.php:141:     * @dataProvider provideEndCases
tests/DocBlock/AnnotationTest.php:166:     * @dataProvider provideTags
tests/DocBlock/AnnotationTest.php:192:     * @dataProvider provideRemoveCases
tests/DocBlock/AnnotationTest.php:220:     * @dataProvider provideTypeParsingCases
tests/DocBlock/AnnotationTest.php:317:     * @dataProvider provideTypesCases
tests/DocBlock/LineTest.php:122:     * @dataProvider provideLines
tests/DocBlock/LineTest.php:135:     * @dataProvider provideLines
tests/DocBlock/LineTest.php:174:     * @dataProvider provideLinesWithUseful
tests/DocBlock/LineTest.php:199:     * @dataProvider provideLinesWithTag
tests/DocBlock/TagComparatorTest.php:34:     * @dataProvider provideComparatorCases
tests/DocBlock/TagTest.php:33:     * @dataProvider provideNameCases
tests/DocBlock/TagTest.php:70:     * @dataProvider provideValidCases
tests/Fixer/Alias/EregToPregFixerTest.php:30:     * @dataProvider provideExamples
tests/Fixer/Alias/MbStrFunctionsFixerTest.php:31:     * @dataProvider provideExamples
tests/Fixer/Alias/NoAliasFunctionsFixerTest.php:31:     * @dataProvider provideCases
tests/Fixer/Alias/NoMixedEchoPrintFixerTest.php:31:     * @dataProvider provideEchoToPrintFixCases
tests/Fixer/Alias/NoMixedEchoPrintFixerTest.php:145:     * @dataProvider providePrintToEchoFixCases
tests/Fixer/Alias/NoMixedEchoPrintFixerTest.php:295:     * @dataProvider provideWrongConfig
tests/Fixer/Alias/PowToExponentiationFixerTest.php:31:     * @dataProvider provideFixCases
tests/Fixer/Alias/PowToExponentiationFixerTest.php:233:     * @dataProvider provideNotFixCases
tests/Fixer/Alias/RandomApiMigrationFixerTest.php:80:     * @dataProvider provideCases
tests/Fixer/ArrayNotation/ArraySyntaxFixerTest.php:64:     * @dataProvider provideLongSyntaxCases
tests/Fixer/ArrayNotation/ArraySyntaxFixerTest.php:104:     * @dataProvider provideShortSyntaxCases
tests/Fixer/ArrayNotation/NoMultilineWhitespaceAroundDoubleArrowFixerTest.php:32:     * @dataProvider provideFixCases
tests/Fixer/ArrayNotation/NoTrailingCommaInSinglelineArrayFixerTest.php:30:     * @dataProvider provideExamples
tests/Fixer/ArrayNotation/NormalizeIndexBraceFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/ArrayNotation/TrailingCommaInMultilineArrayFixerTest.php:30:     * @dataProvider provideExamples
tests/Fixer/ArrayNotation/TrailingCommaInMultilineArrayFixerTest.php:326:     * @dataProvider provideExamples55
tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php:448:     * @dataProvider provideExamples55
tests/Fixer/Basic/BracesFixerTest.php:44:     * @dataProvider provideFixControlContinuationBracesCases
tests/Fixer/Basic/BracesFixerTest.php:726:     * @dataProvider provideFixMissingBracesAndIndentCases
tests/Fixer/Basic/BracesFixerTest.php:2413:     * @dataProvider provideFixClassyBracesCases
tests/Fixer/Basic/BracesFixerTest.php:2564:     * @dataProvider provideFixClassyBraces54Cases
tests/Fixer/Basic/BracesFixerTest.php:2605:     * @dataProvider provideFixAnonFunctionInShortArraySyntax54Cases
tests/Fixer/Basic/BracesFixerTest.php:2692:     * @dataProvider provideFixCommentBeforeBraceCases
tests/Fixer/Basic/BracesFixerTest.php:2768:     * @dataProvider provideFixCommentBeforeBrace70Cases
tests/Fixer/Basic/BracesFixerTest.php:2814:     * @dataProvider provideFixWhitespaceBeforeBraceCases
tests/Fixer/Basic/BracesFixerTest.php:2967:     * @dataProvider provideFixFunctionsCases
tests/Fixer/Basic/BracesFixerTest.php:3284:     * @dataProvider provideFixSpaceAroundTokenCases
tests/Fixer/Basic/BracesFixerTest.php:3913:     * @dataProvider providePreserveLineAfterControlBrace
tests/Fixer/Basic/BracesFixerTest.php:4004:     * @dataProvider provideFixWithAllowOnelineLambdaCases
tests/Fixer/Basic/BracesFixerTest.php:4058:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Basic/BracesFixerTest.php:4129:     * @dataProvider provideDoWhileLoopInsideAnIfWithoutBrackets
tests/Fixer/Basic/EncodingFixerTest.php:31:     * @dataProvider provideExamples
tests/Fixer/Basic/NonPrintableCharacterFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/Basic/Psr0FixerTest.php:205:     * @dataProvider provideIgnoredCases
tests/Fixer/Basic/Psr4FixerTest.php:192:     * @dataProvider provideIgnoredCases
tests/Fixer/Casing/LowercaseConstantsFixerTest.php:31:     * @dataProvider provideGeneratedCases
tests/Fixer/Casing/LowercaseConstantsFixerTest.php:65:     * @dataProvider provideCases
tests/Fixer/Casing/LowercaseKeywordsFixerTest.php:30:     * @dataProvider provideExamples
tests/Fixer/Casing/MagicConstantCasingFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/Casing/MagicConstantCasingFixerTest.php:76:     * @dataProvider provideFixCases54
tests/Fixer/Casing/MagicConstantCasingFixerTest.php:102:     * @dataProvider provideFixCases55
tests/Fixer/Casing/MagicConstantCasingFixerTest.php:128:     * @dataProvider provideFixCases70
tests/Fixer/Casing/NativeFunctionCasingFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/CastNotation/LowercaseCastFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/CastNotation/ModernizeTypesCastingFixerTest.php:31:     * @dataProvider provideExamples
tests/Fixer/CastNotation/NoShortBoolCastFixerTest.php:28:     * @dataProvider provideCases
tests/Fixer/CastNotation/ShortScalarCastFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/CastNotation/ShortScalarCastFixerTest.php:74:     * @dataProvider provideNoFixCases
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:70:     * @dataProvider provideAnonymousClassesCases
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:85:     * @dataProvider provideClassesCases
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:99:     * @dataProvider provideClassesWithConfigCases
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:112:     * @dataProvider provideInterfacesCases
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:125:     * @dataProvider provideTraitsCases
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:412:     * @dataProvider provideClassyDefinitionInfoCases
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:501:     * @dataProvider provideClassyImplementsInfoCases
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:514:     * @dataProvider provideClassyImplementsInfoCases7
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:620:     * @dataProvider providePHP7Cases
tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php:670:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/ClassNotation/MethodSeparationFixerTest.php:31:     * @dataProvider provideCommentBlockStartDetectionCases
tests/Fixer/ClassNotation/MethodSeparationFixerTest.php:125:     * @dataProvider provideFixClassesCases
tests/Fixer/ClassNotation/MethodSeparationFixerTest.php:673:     * @dataProvider provideFixTraitsCases
tests/Fixer/ClassNotation/MethodSeparationFixerTest.php:770:     * @dataProvider provideFixInterfaces
tests/Fixer/ClassNotation/MethodSeparationFixerTest.php:856:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/ClassNotation/NoBlankLinesAfterClassOpeningFixerTest.php:31:     * @dataProvider provideCases
tests/Fixer/ClassNotation/NoBlankLinesAfterClassOpeningFixerTest.php:43:     * @dataProvider provideTraits
tests/Fixer/ClassNotation/NoBlankLinesAfterClassOpeningFixerTest.php:168:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/ClassNotation/NoPhp4ConstructorFixerTest.php:72:     * @dataProvider provideSimpleCases
tests/Fixer/ClassNotation/OrderedClassElementsFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/ClassNotation/OrderedClassElementsFixerTest.php:308:     * @dataProvider provideCases54
tests/Fixer/ClassNotation/OrderedClassElementsFixerTest.php:354:     * @dataProvider provideCases71
tests/Fixer/ClassNotation/OrderedClassElementsFixerTest.php:398:     * @dataProvider provideConfigurationCases
tests/Fixer/ClassNotation/OrderedClassElementsFixerTest.php:444:     * @dataProvider provideConfigurationCases
tests/Fixer/ClassNotation/ProtectedToPrivateFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/ClassNotation/SelfAccessorFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/ClassNotation/SingleClassElementPerStatementFixerTest.php:31:     * @dataProvider provideCases
tests/Fixer/ClassNotation/SingleClassElementPerStatementFixerTest.php:624:     * @dataProvider provideConfigurationCases
tests/Fixer/ClassNotation/SingleClassElementPerStatementFixerTest.php:646:     * @dataProvider provideConfigurationCases
tests/Fixer/ClassNotation/SingleClassElementPerStatementFixerTest.php:724:     * @dataProvider providePHP71Cases
tests/Fixer/ClassNotation/SingleClassElementPerStatementFixerTest.php:761:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/ClassNotation/VisibilityRequiredFixerTest.php:474:     * @dataProvider provideClassConstTest
tests/Fixer/ClassNotation/VisibilityRequiredFixerTest.php:488:     * @dataProvider provideClassConstTest
tests/Fixer/Comment/HashToSlashCommentFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/Comment/HeaderCommentFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/Comment/HeaderCommentFixerTest.php:343:     * @dataProvider provideMisconfiguration
tests/Fixer/Comment/HeaderCommentFixerTest.php:399:     * @dataProvider provideHeaderGenerationCases
tests/Fixer/Comment/HeaderCommentFixerTest.php:442:     * @dataProvider provideFindHeaderCommentInsertionIndexCases
tests/Fixer/Comment/HeaderCommentFixerTest.php:500:     * @dataProvider provideDoNotTouchCases
tests/Fixer/Comment/HeaderCommentFixerTest.php:535:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Comment/NoEmptyCommentFixerTest.php:31:     * @dataProvider provideFixCases
tests/Fixer/Comment/NoEmptyCommentFixerTest.php:208:     * @dataProvider provideCommentBlockCases
tests/Fixer/Comment/NoTrailingWhitespaceInCommentFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/ControlStructure/ElseifFixerTest.php:30:     * @dataProvider provideTestFixCases
tests/Fixer/ControlStructure/NoTrailingCommaInListCallFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/ControlStructure/NoUnneededControlParenthesesFixerTest.php:49:     * @dataProvider provideFixCases
tests/Fixer/ControlStructure/NoUnneededControlParenthesesFixerTest.php:62:     * @dataProvider provideFixCases
tests/Fixer/ControlStructure/NoUnneededControlParenthesesFixerTest.php:75:     * @dataProvider provideFixCases55
tests/Fixer/ControlStructure/NoUnneededControlParenthesesFixerTest.php:89:     * @dataProvider provideFixCases55
tests/Fixer/ControlStructure/NoUnneededControlParenthesesFixerTest.php:103:     * @dataProvider provideFixCases70
tests/Fixer/ControlStructure/NoUnneededControlParenthesesFixerTest.php:117:     * @dataProvider provideFixCases70
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:31:     * @dataProvider providePHPCloseTagCases
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:134:     * @dataProvider provideFixIfElseIfElseCases
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:259:     * @dataProvider provideFixIfElseCases
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:324:     * @dataProvider provideFixNestedIfs
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:365:     * @dataProvider provideBefore54FixCases
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:427:     * @dataProvider provideFixEmptyElseCases
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:521:     * @dataProvider provideNegativeCases
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:694:     * @dataProvider provideBlockDetectionCases
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:756:     * @dataProvider provideConditionsWithoutBraces
tests/Fixer/ControlStructure/NoUselessElseFixerTest.php:847:     * @dataProvider provideIsInConditionWithoutBracesCases
tests/Fixer/ControlStructure/SwitchCaseSemicolonToColonFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/ControlStructure/SwitchCaseSpaceFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/FunctionNotation/FunctionDeclarationFixerTest.php:44:     * @dataProvider provideCases
tests/Fixer/FunctionNotation/FunctionTypehintSpaceFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/FunctionNotation/FunctionTypehintSpaceFixerTest.php:114:     * @dataProvider provideCases56
tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php:266:     * @dataProvider provideFix56Cases
tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php:131:     * @dataProvider provideCasesWithDefaultConfiguration
tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php:201:     * @dataProvider provideCasesWithConfiguredExclude
tests/Fixer/FunctionNotation/NoUnreachableDefaultArgumentValueFixerTest.php:25:     * @dataProvider provideCases
tests/Fixer/Import/NoLeadingImportSlashFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/Import/NoLeadingImportSlashFixerTest.php:41:     * @dataProvider provideFix54Cases
tests/Fixer/Import/NoUnusedImportsFixerTest.php:559:     * @dataProvider provideCloseTagCases
tests/Fixer/Import/OrderedImportsFixerTest.php:611:     * @dataProvider provideCommentCases
tests/Fixer/Import/OrderedImportsFixerTest.php:818:     * @dataProvider provideInvalidSortAlgorithmConfiguration
tests/Fixer/Import/SingleImportPerStatementFixerTest.php:32:     * @dataProvider provideCases
tests/Fixer/Import/SingleImportPerStatementFixerTest.php:298:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Import/SingleLineAfterImportsFixerTest.php:31:     * @dataProvider provideCases
tests/Fixer/Import/SingleLineAfterImportsFixerTest.php:435:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/LanguageConstruct/ClassKeywordRemoveFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/LanguageConstruct/ClassKeywordRemoveFixerTest.php:210:     * @dataProvider provideFixCases70
tests/Fixer/LanguageConstruct/CombineConsecutiveUnsetsFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/LanguageConstruct/DeclareEqualNormalizeFixerTest.php:45:     * @dataProvider provideCases
tests/Fixer/LanguageConstruct/DeclareEqualNormalizeFixerTest.php:112:     * @dataProvider provideInvalidConfig
tests/Fixer/LanguageConstruct/DirConstantFixerTest.php:31:     * @dataProvider provideExamples
tests/Fixer/LanguageConstruct/FunctionToConstantFixerTest.php:31:     * @dataProvider provideTestCases
tests/Fixer/LanguageConstruct/FunctionToConstantFixerTest.php:142:     * @dataProvider provideInvalidConfiguration
tests/Fixer/LanguageConstruct/IsNullFixerTest.php:59:     * @dataProvider provideExamples
tests/Fixer/LanguageConstruct/SilencedDeprecationErrorFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/NamespaceNotation/BlankLineAfterNamespaceFixerTest.php:31:     * @dataProvider provideCases
tests/Fixer/NamespaceNotation/BlankLineAfterNamespaceFixerTest.php:148:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/NamespaceNotation/NoBlankLinesBeforeNamespaceFixerTest.php:28:     * @dataProvider provideExamples
tests/Fixer/NamespaceNotation/NoLeadingNamespaceWhitespaceFixerTest.php:32:     * @dataProvider provideExamples
tests/Fixer/NamespaceNotation/NoLeadingNamespaceWhitespaceFixerTest.php:147:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/NamespaceNotation/SingleBlankLineBeforeNamespaceFixerTest.php:28:     * @dataProvider provideExamples
tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php:36:     * @dataProvider provideCases
tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php:212:     * @dataProvider provideUnalignEqualsCases
tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php:375:     * @dataProvider provideCases54
tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php:407:     * @dataProvider provideUnalignDoubleArrowCases
tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php:804:     * @dataProvider provideAlignEqualsCases
tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php:930:     * @dataProvider provideAlignDoubleArrowCases
tests/Fixer/Operator/ConcatSpaceFixerTest.php:51:     * @dataProvider provideWithoutSpaceCases
tests/Fixer/Operator/ConcatSpaceFixerTest.php:153:     * @dataProvider provideWithSpaceCases
tests/Fixer/Operator/NewWithBracesFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Operator/NotOperatorWithSpaceFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Operator/NotOperatorWithSuccessorSpaceFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Operator/ObjectOperatorWithoutWhitespaceFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Operator/PreIncrementFixerTest.php:30:     * @dataProvider provideExamples
tests/Fixer/Operator/StandardizeNotEqualsFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Operator/TernaryOperatorSpacesFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Operator/TernaryToNullCoalescingFixerTest.php:28:     * @dataProvider provideCases
tests/Fixer/Operator/UnaryOperatorSpacesFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Operator/UnaryOperatorSpacesFixerTest.php:112:     * @dataProvider provideCasesLT54
tests/Fixer/Operator/UnaryOperatorSpacesFixerTest.php:134:     * @dataProvider provideCases56
tests/Fixer/PhpTag/BlankLineAfterOpeningTagFixerTest.php:32:     * @dataProvider provideCases
tests/Fixer/PhpTag/BlankLineAfterOpeningTagFixerTest.php:44:     * @dataProvider provideCases54
tests/Fixer/PhpTag/BlankLineAfterOpeningTagFixerTest.php:134:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/PhpTag/FullOpeningTagFixerTest.php:28:     * @dataProvider provideFixCases
tests/Fixer/PhpTag/FullOpeningTagFixerTest.php:112:     * @dataProvider provideFixCasesLT70
tests/Fixer/PhpTag/LinebreakAfterOpeningTagFixerTest.php:32:     * @dataProvider provideCases
tests/Fixer/PhpTag/LinebreakAfterOpeningTagFixerTest.php:44:     * @dataProvider provideCases54
tests/Fixer/PhpTag/LinebreakAfterOpeningTagFixerTest.php:109:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/PhpTag/NoClosingTagFixerTest.php:28:     * @dataProvider provideCasesWithFullOpenTag
tests/Fixer/PhpTag/NoClosingTagFixerTest.php:39:     * @dataProvider provideCasesWithShortOpenTag
tests/Fixer/PhpTag/NoShortEchoTagFixerTest.php:30:     * @dataProvider provideClosingTagExamples
tests/Fixer/PhpUnit/PhpUnitConstructFixerTest.php:31:     * @dataProvider provideTestFixCases
tests/Fixer/PhpUnit/PhpUnitConstructFixerTest.php:57:     * @dataProvider provideTestFixCases
tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php:30:     * @dataProvider provideInternalTypeMethods
tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php:114:     * @dataProvider provideDedicatedAssertsCases
tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php:206:     * @dataProvider provideNotFixCases
tests/Fixer/PhpUnit/PhpUnitStrictFixerTest.php:31:     * @dataProvider provideTestFixCases
tests/Fixer/PhpUnit/PhpUnitStrictFixerTest.php:49:     * @dataProvider provideTestFixCases
tests/Fixer/Phpdoc/NoBlankLinesAfterPhpdocFixerTest.php:243:     * @dataProvider provideInlineTypehintingDocsBeforeFlowBreakCases
tests/Fixer/Phpdoc/NoEmptyPhpdocFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/Phpdoc/PhpdocAddMissingParamAnnotationFixerTest.php:78:     * @dataProvider provideFixCases
tests/Fixer/Phpdoc/PhpdocAddMissingParamAnnotationFixerTest.php:297:     * @dataProvider provideCases70
tests/Fixer/Phpdoc/PhpdocAddMissingParamAnnotationFixerTest.php:338:     * @dataProvider provideCases71
tests/Fixer/Phpdoc/PhpdocAddMissingParamAnnotationFixerTest.php:372:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php:452:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php:530:     * @dataProvider provideVariadicCases
tests/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php:30:     * @dataProvider provideDocblocks
tests/Fixer/Phpdoc/PhpdocInlineTagFixerTest.php:30:     * @dataProvider provideTestFixInlineDocCases
tests/Fixer/Phpdoc/PhpdocInlineTagFixerTest.php:165:     * @dataProvider provideTestFixInheritDocCases
tests/Fixer/Phpdoc/PhpdocNoAliasTagFixerTest.php:104:     * @dataProvider providePropertyCases
tests/Fixer/Phpdoc/PhpdocNoAliasTagFixerTest.php:121:     * @dataProvider providePropertyCases
tests/Fixer/Phpdoc/PhpdocNoAliasTagFixerTest.php:163:     * @dataProvider provideTypeToVarCases
tests/Fixer/Phpdoc/PhpdocNoAliasTagFixerTest.php:224:     * @dataProvider provideVarToTypeCases
tests/Fixer/Phpdoc/PhpdocNoUselessInheritdocFixerTest.php:30:     * @dataProvider provideDoFixCases
tests/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixerTest.php:31:     * @dataProvider provideDefaultConfigurationTestCases
tests/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixerTest.php:44:     * @dataProvider provideDefaultConfigurationTestCases
tests/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixerTest.php:78:     * @dataProvider provideTestCases
tests/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixerTest.php:92:     * @dataProvider provideTestCases
tests/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixerTest.php:115:     * @dataProvider provideGeneratedFixCases
tests/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixerTest.php:182:     * @dataProvider provideInvalidConfiguration
tests/Fixer/Phpdoc/PhpdocSingleLineVarSpacingFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Phpdoc/PhpdocSummaryFixerTest.php:376:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Phpdoc/PhpdocToCommentFixerTest.php:30:     * @dataProvider provideDocblocks
tests/Fixer/Phpdoc/PhpdocToCommentFixerTest.php:42:     * @dataProvider provideTraits
tests/Fixer/Phpdoc/PhpdocToCommentFixerTest.php:562:     * @dataProvider provideCases71
tests/Fixer/ReturnNotation/BlankLineBeforeReturnFixerTest.php:31:     * @dataProvider provideCases
tests/Fixer/ReturnNotation/BlankLineBeforeReturnFixerTest.php:152:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/ReturnNotation/NoUselessReturnFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/ReturnNotation/SimplifiedNullReturnFixerTest.php:30:     * @dataProvider provideExamples
tests/Fixer/Semicolon/NoEmptyStatementFixerTest.php:31:     * @dataProvider provideNoEmptyStatementsCases
tests/Fixer/Semicolon/NoEmptyStatementFixerTest.php:434:     * @dataProvider providePHP7Cases
tests/Fixer/Semicolon/NoEmptyStatementFixerTest.php:529:     * @dataProvider provideCasesWithShortOpenTag
tests/Fixer/Semicolon/NoEmptyStatementFixerTest.php:572:     * @dataProvider provideFixMultipleSemicolonsCases
tests/Fixer/Semicolon/NoMultilineWhitespaceBeforeSemicolonsFixerTest.php:33:     * @dataProvider provideCases
tests/Fixer/Semicolon/NoMultilineWhitespaceBeforeSemicolonsFixerTest.php:131:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Semicolon/NoSinglelineWhitespaceBeforeSemicolonsFixerTest.php:32:     * @dataProvider provideCases
tests/Fixer/Semicolon/SemicolonAfterInstructionFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Semicolon/SpaceAfterSemicolonFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Strict/DeclareStrictTypesFixerTest.php:31:     * @dataProvider provideFixCases
tests/Fixer/Strict/DeclareStrictTypesFixerTest.php:111:     * @dataProvider provideDoNotFixCases
tests/Fixer/Strict/DeclareStrictTypesFixerTest.php:130:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Strict/StrictComparisonFixerTest.php:30:     * @dataProvider provideTestFixCases
tests/Fixer/Strict/StrictParamFixerTest.php:30:     * @dataProvider provideFixCases
tests/Fixer/StringNotation/HeredocToNowdocFixerTest.php:30:     * @dataProvider provideTestFixCases
tests/Fixer/StringNotation/SingleQuoteFixerTest.php:30:     * @dataProvider provideTestFixCases
tests/Fixer/Whitespace/IndentationTypeFixerTest.php:31:     * @dataProvider provideFixCases
tests/Fixer/Whitespace/IndentationTypeFixerTest.php:231:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Whitespace/LineEndingFixerTest.php:32:     * @dataProvider provideCases
tests/Fixer/Whitespace/LineEndingFixerTest.php:70:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:108:     * @dataProvider provideConfigTests
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:122:     * @dataProvider provideConfigTests
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:386:     * @dataProvider provideCommentCases
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:460:     * @dataProvider provideBetweenUseCases
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:565:     * @dataProvider provideRemoveLinesBetweenUseStatements70Cases
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:597:     * @dataProvider provideWithoutUsesCases
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:728:     * @dataProvider provideOneAndInLineCases
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:768:     * @dataProvider provideOneAndInLine70Cases
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:799:     * @dataProvider provideBraceCases
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:895:     * @dataProvider provideBraceCases54
tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php:919:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Whitespace/NoSpacesAroundOffsetFixerTest.php:30:     * @dataProvider provideInsideCases
tests/Fixer/Whitespace/NoSpacesAroundOffsetFixerTest.php:41:     * @dataProvider provideOutsideCases
tests/Fixer/Whitespace/NoSpacesAroundOffsetFixerTest.php:74:     * @dataProvider provideCommentCases
tests/Fixer/Whitespace/NoSpacesAroundOffsetFixerTest.php:301:     * @dataProvider provideConfigurationCases
tests/Fixer/Whitespace/NoSpacesAroundOffsetFixerTest.php:314:     * @dataProvider provideConfigurationCases
tests/Fixer/Whitespace/NoSpacesAroundOffsetFixerTest.php:382:     * @dataProvider providePHP71Cases
tests/Fixer/Whitespace/NoSpacesInsideParenthesisFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Whitespace/NoTrailingWhitespaceFixerTest.php:30:     * @dataProvider provideCases
tests/Fixer/Whitespace/NoWhitespaceInBlankLineFixerTest.php:31:     * @dataProvider provideCases
tests/Fixer/Whitespace/NoWhitespaceInBlankLineFixerTest.php:150:     * @dataProvider provideMessyWhitespacesCases
tests/Fixer/Whitespace/SingleBlankLineAtEofFixerTest.php:31:     * @dataProvider provideCases
tests/Fixer/Whitespace/SingleBlankLineAtEofFixerTest.php:150:     * @dataProvider provideMessyWhitespacesCases
tests/FixerFactoryTest.php:210:     * @dataProvider provideConflictingFixersRules
tests/FixerNameValidatorTest.php:32:     * @dataProvider provideIsValidCases
tests/Linter/AbstractLinterTestCase.php:31:     * @dataProvider provideLintFileCases
tests/Linter/AbstractLinterTestCase.php:64:     * @dataProvider provideLintSourceCases
tests/RuleSetTest.php:42:     * @dataProvider provideAllRulesFromSets
tests/RuleSetTest.php:275:     * @dataProvider provideSafeSets
tests/RuleSetTest.php:351:     * @dataProvider provideResolveRulesCases
tests/TextDiffTest.php:35:     * @dataProvider provideDiffReporting
tests/Tokenizer/Analyzer/ArgumentsAnalyzerTest.php:34:     * @dataProvider provideCases
tests/Tokenizer/CTTest.php:36:     * @dataProvider provideCTs
tests/Tokenizer/TokenTest.php:75:     * @dataProvider provideIsCastCases
tests/Tokenizer/TokenTest.php:101:     * @dataProvider provideIsClassyCases
tests/Tokenizer/TokenTest.php:128:     * @dataProvider provideIsCommentCases
tests/Tokenizer/TokenTest.php:186:     * @dataProvider provideMagicConstantCases
tests/Tokenizer/TokenTest.php:226:     * @dataProvider provideIsNativeConstantCases
tests/Tokenizer/TokenTest.php:251:     * @dataProvider provideIsWhitespaceCases
tests/Tokenizer/TokenTest.php:285:     * @dataProvider provideCreatingTokenCases
tests/Tokenizer/TokenTest.php:322:     * @dataProvider provideEquals
tests/Tokenizer/TokenTest.php:378:     * @dataProvider provideEqualsAny
tests/Tokenizer/TokenTest.php:409:     * @dataProvider provideIsKeyCaseSensitive
tests/Tokenizer/TokensAnalyzerTest.php:175:     * @dataProvider provideIsAnonymousClassCases
tests/Tokenizer/TokensAnalyzerTest.php:215:     * @dataProvider provideIsLambdaCases
tests/Tokenizer/TokensAnalyzerTest.php:266:     * @dataProvider provideIsLambdaCases70
tests/Tokenizer/TokensAnalyzerTest.php:301:     * @dataProvider provideIsLambdaCases71
tests/Tokenizer/TokensAnalyzerTest.php:350:     * @dataProvider provideIsUnarySuccessorOperator
tests/Tokenizer/TokensAnalyzerTest.php:406:     * @dataProvider provideIsUnaryPredecessorOperator
tests/Tokenizer/TokensAnalyzerTest.php:470:     * @dataProvider provideIsUnaryPredecessorOperator56
tests/Tokenizer/TokensAnalyzerTest.php:515:     * @dataProvider provideIsBinaryOperator
tests/Tokenizer/TokensAnalyzerTest.php:635:     * @dataProvider provideIsBinaryOperator56
tests/Tokenizer/TokensAnalyzerTest.php:668:     * @dataProvider provideIsBinaryOperator70
tests/Tokenizer/TokensAnalyzerTest.php:703:     * @dataProvider provideIsArrayCases
tests/Tokenizer/TokensAnalyzerTest.php:784:     * @dataProvider provideIsArray71Cases
tests/Tokenizer/TokensAnalyzerTest.php:821:     * @dataProvider provideArrayExceptions
tests/Tokenizer/TokensAnalyzerTest.php:834:     * @dataProvider provideArrayExceptions
tests/Tokenizer/TokensAnalyzerTest.php:862:     * @dataProvider provideGetFunctionProperties
tests/Tokenizer/TokensTest.php:53:     * @dataProvider provideFindSequence
tests/Tokenizer/TokensTest.php:283:     * @dataProvider provideFindSequenceExceptions
tests/Tokenizer/TokensTest.php:346:     * @dataProvider provideMonolithicPhpDetection
tests/Tokenizer/TokensTest.php:372:     * @dataProvider provideShortOpenTagMonolithicPhpDetection
tests/Tokenizer/TokensTest.php:408:     * @dataProvider provideShortOpenTagEchoMonolithicPhpDetection
tests/Tokenizer/TokensTest.php:638:     * @dataProvider provideTokenOfKindSiblingCases
tests/Tokenizer/TokensTest.php:703:     * @dataProvider provideFindBlockEndCases
tests/Tokenizer/TokensTest.php:731:     * @dataProvider provideFindBlockEndCases71
tests/Tokenizer/TokensTest.php:825:     * @dataProvider provideIsEmptyCases
tests/Tokenizer/TokensTest.php:898:     * @dataProvider provideEnsureWhitespaceAtIndexCases
tests/Tokenizer/Transformer/ArrayTypehintTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/BraceClassInstantiationTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/ClassConstantTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/CurlyBraceTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/CurlyBraceTransformerTest.php:57:     * @dataProvider provideProcessCases70
tests/Tokenizer/Transformer/ImportTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/NamespaceOperatorTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/NullableTypeTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/ReturnRefTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/SquareBraceTransformerTest.php:37:     * @dataProvider provideIsShortArrayCases
tests/Tokenizer/Transformer/SquareBraceTransformerTest.php:87:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/SquareBraceTransformerTest.php:230:     * @dataProvider provideProcessCases71
tests/Tokenizer/Transformer/SquareBraceTransformerTest.php:302:     * @dataProvider provideProcessCases72
tests/Tokenizer/Transformer/TypeAlternationTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/TypeColonTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/UseTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/Tokenizer/Transformer/WhitespacyCommentTransformerTest.php:30:     * @dataProvider provideProcessCases
tests/UtilsTest.php:34:     * @dataProvider provideCamelCaseToUnderscoreCases
tests/UtilsTest.php:74:     * @dataProvider provideCmpIntCases
tests/UtilsTest.php:97:     * @dataProvider provideSplitLinesCases
tests/UtilsTest.php:130:     * @dataProvider provideCalculateTrailingWhitespaceIndentCases
tests/WhitespacesFixerConfigTest.php:32:     * @dataProvider provideTestCases
```

### Prefix `provider`

```
$ git grep "@dataProvider provider"

tests/Cache/CacheTest.php:118:     * @dataProvider providerMissingData
tests/Doctrine/Annotation/TokenTest.php:62:     * @dataProvider providerIsType
tests/Doctrine/Annotation/TokenTest.php:97:     * @dataProvider providerIsNotType
tests/Fixer/ArrayNotation/NoWhitespaceBeforeCommaInArrayFixerTest.php:30:     * @dataProvider providerFix
tests/Fixer/ArrayNotation/WhitespaceAfterCommaInArrayFixerTest.php:30:     * @dataProvider providerFix
tests/Fixer/CastNotation/CastSpacesFixerTest.php:28:     * @dataProvider providerFixCasts
tests/Fixer/ControlStructure/IncludeFixerTest.php:31:     * @dataProvider providerFix
tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php:32:     * @dataProvider providerFix
tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php:41:     * @dataProvider providerInvalidConfigurationElement
tests/Fixer/FunctionNotation/NoSpacesAfterFunctionNameFixerTest.php:30:     * @dataProvider providerFix
tests/Fixer/FunctionNotation/ReturnTypeDeclarationFixerTest.php:39:     * @dataProvider providerFixWithSpaceBeforeNone
tests/Fixer/FunctionNotation/ReturnTypeDeclarationFixerTest.php:53:     * @dataProvider providerFixWithSpaceBeforeNone
tests/Fixer/FunctionNotation/ReturnTypeDeclarationFixerTest.php:66:     * @dataProvider providerFixWithSpaceBeforeNone
tests/Fixer/FunctionNotation/ReturnTypeDeclarationFixerTest.php:135:     * @dataProvider providerFixWithSpaceBeforeOne
tests/Fixer/Import/NoUnusedImportsFixerTest.php:453:     * @dataProvider providerUseInString
tests/Fixer/Phpdoc/PhpdocAddMissingParamAnnotationFixerTest.php:43:     * @dataProvider providerInvalidConfigurationValue
tests/FixerDefinition/VersionSpecificCodeSampleTest.php:57:     * @dataProvider providerIsSuitableForVersionUsesVersionSpecification
tests/FixerDefinition/VersionSpecificationTest.php:35:     * @dataProvider providerInvalidVersion
tests/FixerDefinition/VersionSpecificationTest.php:47:     * @dataProvider providerInvalidVersion
tests/FixerDefinition/VersionSpecificationTest.php:88:     * @dataProvider providerIsSatisfiedByReturnsTrue
tests/FixerDefinition/VersionSpecificationTest.php:118:     * @dataProvider providerIsSatisfiedByReturnsFalse
tests/RuleSetTest.php:96:     * @dataProvider providerSetDefinitionNames
tests/RuleSetTest.php:229:     * @dataProvider providerSetDefinitionNames
```

### Other Prefix 

```
$ git grep "@dataProvider [^p]"

tests/AbstractDoctrineAnnotationFixerTestCase.php:27:     * @dataProvider getInvalidConfigurationCases
tests/AutoReview/FixerFactoryTest.php:41:     * @dataProvider getFixersPriorityCases
tests/CiIntegrationTest.php:90:     * @dataProvider getIntegrationCases
tests/Console/ConfigurationResolverTest.php:128:     * @dataProvider getProgressTypeCases
tests/Console/ConfigurationResolverTest.php:150:     * @dataProvider getProgressTypeCases
tests/Console/ConfigurationResolverTest.php:700:     * @dataProvider getResolveBooleanOptions
tests/Console/ConfigurationResolverTest.php:805:     * @dataProvider getResolveBooleanOptions
tests/Console/Output/ProcessOutputTest.php:31:     * @dataProvider getProcessProgressOutputCases
tests/Console/Output/ProcessOutputTest.php:103:     * @dataProvider getProcessProgressOutputWithNumbersCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationBracesFixerTest.php:29:     * @dataProvider getFixWithBracesCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationBracesFixerTest.php:271:     * @dataProvider getFixWithoutBracesCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationIndentationFixerTest.php:29:     * @dataProvider getFixCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:29:     * @dataProvider getFixAllCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:48:     * @dataProvider getFixAllCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:279: * @dataProvider ( foo,bar  =  "baz" )
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:328:     * @dataProvider getFixAroundParenthesesOnlyCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:352:     * @dataProvider getFixAroundParenthesesOnlyCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:538: * @dataProvider ( foo,bar  =  "baz" )
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:583:     * @dataProvider getFixAroundCommasOnlyCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:607:     * @dataProvider getFixAroundCommasOnlyCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:816: * @dataProvider ( foo,bar  =  "baz" )
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:861:     * @dataProvider getFixAroundArgumentAssignmentsOnlyCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:885:     * @dataProvider getFixAroundArgumentAssignmentsOnlyCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:1086: * @dataProvider ( foo,bar  =  "baz" )
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:1131:     * @dataProvider getFixAroundArrayAssignmentsOnlyCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:1155:     * @dataProvider getFixAroundArrayAssignmentsOnlyCases
tests/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixerTest.php:1336: * @dataProvider ( foo,bar  =  "baz" )
tests/Test/AbstractIntegrationTestCase.php:106:     * @dataProvider getTests
tests/Tokenizer/TokensAnalyzerTest.php:1009:     * @dataProvider getImportUseIndexesCases
tests/Tokenizer/TokensAnalyzerTest.php:1084:     * @dataProvider getImportUseIndexesCasesPHP70
tests/Tokenizer/TokensTest.php:533:     * @dataProvider getClearTokenAndMergeSurroundingWhitespaceCases
```

Reply:
Maybe we can start with safe guarding new provides without the prefix are added and rename the other in a new major to prevent merging-issues? IDK, like the idea though ;)

Reply:
for most cases we do have `testSomeMethod` and `provideSomeMethodCases`, only extra old providers are not following this pattern

Reply:
actually, why some providers were already renamed? :)

Reply:
ah, extracted already to new PR ! so great ;)

Reply:
:+1: